src/Calculator/SettingsPage.xaml

```diff
@@ -0,0 +1,164 @@
+        <Style x:Key="CategoryNameTextBlockStyle" TargetType="TextBlock">
+            <Setter Property="FontSize" Value="20"/>
+            <Setter Property="FontWeight" Value="SemiBold"/>
+            <Setter Property="OpticalMarginAlignment" Value="TrimSideBearings"/>
+        </Style>
+    </Page.Resources>
+
+    <ScrollViewer>
+        <Grid x:Name="PageGrid" AutomationProperties.LandmarkType="Main">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="1*"
+                           MinHeight="{StaticResource HamburgerHeight}"
+                           MaxHeight="52"/>
+                <RowDefinition Height="1*"/>
+            </Grid.RowDefinitions>
+
+            <Grid Height="{StaticResource HamburgerHeight}"
+              HorizontalAlignment="Left"
+              VerticalAlignment="Top">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="{StaticResource HamburgerHeightGridLength}"/>
+                    <ColumnDefinition Width="Auto"/>
+                </Grid.ColumnDefinitions>
+
+                <Button x:Name="BackButton"
+                    x:Uid="BackButton"
+                    HorizontalAlignment="Stretch"
+                    VerticalAlignment="Stretch"
+                    Background="{ThemeResource AppChromeAcrylicHostBackdropMediumLowBrush}"
+                    FontFamily="Segoe MDL2 Assets"
+                    FontSize="15"
+                    Windows10version1809:CornerRadius="0"
+                    Click="BackButtonClick"
+                    Content="&#xE72B;"/>
+
+                <TextBlock x:Uid="SettingsHeader"
+                       Grid.Column="1"
+                       Margin="8,-3,0,0"
+                       HorizontalAlignment="Left"
+                       VerticalAlignment="Center"
+                       Style="{StaticResource CategoryNameTextBlockStyle}"/>
+            </Grid>
+
+            <StackPanel Grid.Row="1" Margin="12,0,12,0">
+                <StackPanel x:Name="SettingsStack"
+                        AutomationProperties.LabeledBy="{Binding ElementName=SettingsAppTheme}"
+                        Orientation="Vertical">
+
+                    <TextBlock x:Name="SettingsAppTheme"
+                           x:Uid="SettingsAppTheme"
+                           Style="{StaticResource SubtitleTextBlockStyle}"
+                           AutomationProperties.HeadingLevel="Level1"
+                           TextTrimming="Clip"/>
+
+                    <RadioButton x:Name="SettingsLightTheme"
+                             x:Uid="SettingsLightTheme"
+                             Grid.Row="3"
+                             Checked="LightChecked"/>
+
+                    <RadioButton x:Name="SettingsDarkTheme"
+                             x:Uid="SettingsDarkTheme"
+                             Grid.Row="2"
+                             Checked="DarkChecked"/>
+
+                    <RadioButton x:Name="SettingsSystemTheme"
```

Comment:
If I set this radio button, hit the back button, and then go back to settings, my selection is reset. I think you may be missing code to load from settings on page navigation.